session1/solutions/assignment4/Rakefile

```diff
@@ -1 +1,42 @@
 # assignment 4 solution
+ #namespace :gziped_files do
```

Comment:
Given this input: `rake archive[/tmp/folder,.tmp]`, I received the following error:

rake aborted!
TypeError: no implicit conversion of nil into String
/Users/eyals/dev/coding-assignments/session1/Rakefile:12:in `chdir'
/Users/eyals/dev/coding-assignments/session1/Rakefile:12:in `block in <top (required)>'
Tasks: TOP => archive
(See full trace by running task with --trace)

Reply:
I did it like in the Example :
Calling the command "rake archive PATH=/home/user/my_archive EXCLUDE=.tmp,.bkp" 
see the screen shot
![2018-01-10_11-20-16](https://user-images.githubusercontent.com/12675378/34764900-8b07293a-f5f8-11e7-8e91-3474cf9addc2.jpg)

do i need to change the code to the syntax of the input that you tried?